From d2b051b352a5f00926d4396c0115b2e04ceb87a6 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Tue, 20 Oct 2015 14:33:19 +0800
Subject: [PATCH 290/290] drm: xylon: add missed default set_busid to avoid
 warnings

This is to set busid as the default one xilinx, and avoid
the following warnings:

WARNING: CPU: 0 PID: 344 at drivers/gpu/drm/drm_ioctl.c:144 drm_setversion+0xd)
No drm_driver.set_busid() implementation provided by 0x809c8b44.
Use drm_dev_set_unique() to set the unique name explicitly.
Modules linked in:
 zynq_edac edac_core
CPU: 1 PID: 344 Comm: Xorg Not tainted 4.1.10-WR8.0.0.0_standard #2
Hardware name: Xilinx Zynq Platform
(unwind_backtrace) from [<800136b4>] (show_stack+0x20/0x24)
(show_stack) from [<806706a4>] (dump_stack+0x80/0xcc)
(dump_stack) from [<80027b30>] (warn_slowpath_common+0x98/0xc4)
(warn_slowpath_common) from [<80027b9c>] (warn_slowpath_fmt+0x40/0x48)
(warn_slowpath_fmt) from [<803febe4>] (drm_setversion+0xd4/0x16c)
(drm_setversion) from [<803fe81c>] (drm_ioctl+0x3a4/0x444)
(drm_ioctl) from [<8014e1d0>] (do_vfs_ioctl+0x504/0x5c4)
(do_vfs_ioctl) from [<8014e2ec>] (SyS_ioctl+0x5c/0x84)
(SyS_ioctl) from [<8000f3a0>] (ret_fast_syscall+0x0/0x3c)

Signed-off-by:Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/xylon/xylon_drv.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/xylon/xylon_drv.c b/drivers/gpu/drm/xylon/xylon_drv.c
index 0e6de30..c02f612 100644
--- a/drivers/gpu/drm/xylon/xylon_drv.c
+++ b/drivers/gpu/drm/xylon/xylon_drv.c
@@ -243,6 +243,7 @@ static struct drm_driver xylon_drm_driver = {
 	.preclose = xylon_drm_preclose,
 	.postclose = xylon_drm_postclose,
 	.lastclose = xylon_drm_lastclose,
+	.set_busid = drm_platform_set_busid,
 
 	.get_vblank_counter = drm_vblank_count,
 	.enable_vblank = xylon_drm_vblank_enable,
-- 
2.0.2

